Suppose a race is taking place with **five horses**. Nothing is known about them beforehand and betting is opened as to who will win. Suppose that **50 people place bets**. 

The betting company want to know if any cheating is taking place – that is, **if more people correctly guess the winner and make money than purely due to chance.**

1. **What is the null hypothesis?**

2. Find the **p-value if 12 people win money** – what does this mean for the null hypothesis (using alpha = 0.05)?

1. Null hypothesis is that no cheating is taking place and the amount of winners is due to random chance.

2. n = 50, 1/5 chance of winning or 0.2.  so that would be 10 people winning
actual result is that 12 people win so slightly over expected. Calculating a p-value will help to determine how likely this is due to random chance or if it is more likely that cheating is taking place

The p-value is calculated to be 0.4046 which is well above the alpha value of 0.05 so we fail to rejuct the null hypothesis that no cheating is taking place and the results are due to chance. 

In [11]:
import scipy

n = 50
chance_of_winning = 1/5
pop_mean_wins = n*chance_of_winning
sample_wins = 12

results = np.array([[sample_wins, n - sample_wins], [pop_mean_wins, n - pop_mean_wins]])
chi2, p_value, df, _ = scipy.stats.chi2_contingency(results)

print(f'p-value for single sided test: {p_value / 2:.4f}')

p-value for single sided test: 0.4046


In [12]:
p = scipy.stats.binom_test(x=12, n=50, p=0.2, alternative='greater')

p

0.28933239501182995

In [2]:
import numpy as np
from scipy.stats import ttest_ind

v1 = np.random.normal(size=100)
v2 = np.random.normal(size=100)

res = ttest_ind(v1,v2)
print(res)

Ttest_indResult(statistic=0.7961736694961222, pvalue=0.42688463608002614)


In [1]:
from scipy import stats
rvs = stats.norm.rvs(loc = 5, scale = 10, size = (50,2))
print(stats.ttest_1samp(rvs,5.0))

Ttest_1sampResult(statistic=array([1.14436344, 1.95224402]), pvalue=array([0.25803236, 0.05663531]))


with a p-value of 0.4046 we fail to reject the null hypothesis that no cheating is taking place and that the amount of winners is due to random chance with an alpha = 0.05

# Tutor Session Lecture Notes

Assuming IID - identically and independentaly distributed - expect 10 people to win
Binomial Plots  
P(X=r) = nCr*p^r*(1-p)^n-r  
## p-value of 0.289  => same as my stats.binom_test

model relies on horses having equal chances of winning and everyone thinking they have same chance of winning

Why model might not be correct?   

independent drawing from the samples - no ones choice of hores affects another - not realistic in hores racing  
go to the track with friends they will be swayed by each other

Why not Normal distribution?  
count data - whole numbers - non-continuous and non negative values  
has to stop at 0 so there is a backstop to the distribution at 0  
